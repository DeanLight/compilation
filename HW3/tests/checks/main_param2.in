void main (int param)
{
	return;
}
void main()
{
	return;
}