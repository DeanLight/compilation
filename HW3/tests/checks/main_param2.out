---end scope---
param INT -1
line 5: identifier main is already defined
